.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Build project with Maven
+on:
+  pull_request:
+  push:
+  schedule:
+  - cron: '2 2 * * 1-5' # run nightly master builds on weekdays
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    services:
+      keycloak:
+        image: jboss/keycloak:8.0.2
```

Comment:
Great job. 👍 
One question: would you mind moving to `jboss/keycloak:10.0.1` as configured in the current CI snippet?

Reply:
speaking of the CI snippet: I think we can get rid of this script now, as we're removing this repo from our Jenkins once this PR is merged.

@VonDerBeck do you see a reason to keep (and maintain) it?

Reply:
I like the way you think! Great working with you :cowboy_hat_face: